bundle/tests/scorecard/kuttl/monitor/02-assert.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 60
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: service-monitor-rc
+status:
+  readyReplicas: 1
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: service-monitor-rc
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: service-monitor-rc
+  endpoints:
+  - path: "/path"
+    scheme: "myScheme"
+    port: 3000-tcp
+    params:
+      params:
+      - param1
+      - param2
+    scrapeTimeout: 10s
+    interval: 30s
+    bearerTokenFile: myBTF
+    tlsConfig:
+      insecureSkipVerify: true
+    basicAuth:
+      password:
+        key: password
+      username:
+        key: username
+---
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
```

Comment:
What is the purpose of this RuntimeComponent snippet in assert?

Reply:
Not sure, that looks like a mistake. It is only asserting on the spec, which should never be incorrect, so I can't see that there is a point. I will take that out.